schemas/derivatives/SCHEMA.md

```diff
@@ -0,0 +1,83 @@
+| protocol_fees_usd        | (Optional) The portion of fees accrued to the protocol, in USD. | number |
+
+### Pool Snapshot
+
+Snapshot of the pool's metrics.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| chain_id                 | Standard chain id.                                        | number |
+| pool_address             | The smart contract address of the pool.                   | string |
+| funding_rate             | The funding rate in this pool at the time of the snapshot, as a percentage. | number |
+| fee_rate                 | he pool's fee rate, as a percentage.                      | number |
+| total_value_locked_usd   | The total value locked in USD.                            | number |
+| volume_usd               | The volume of positions opened and closed in the given snapshot in USD. | number |
+| open_interest_longs_usd  | The sum of open interest of longs held within this pool in USD. | number |
+| open_interest_shorts_usd | The sum of open interest in shorts held within this pool in USD. | number |
+
+### Trades
+
+Trade data, 1 entry for each close, open, or update of a trade.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the trade.                               | number |
+| chain_id                 | Standard chain id.                                        | number |
+| transaction_hash         | The hash of the transaction this trade was performed in.  | string |
+| block_number             | The block number of the trade.                            | number |
+| pool_address             | The address of the pool this token was traded in.         | string |
+| maker_address            | The address of the maker.                                 | string |
+| taker_address            | The address of the taker.                                 | string |
+| token_address            | The address of the token.                                 | string |
+| amount                   | The value that a trader is putting to open a long or short, decimal normalized. | number |
+| amount_usd               | The trade amount in USD.                                  | number |
+| notional_value           | The value of the leveraged amount of the trade, decimal normalized. | number |
+| notional_value_usd       | The notional value, in USD.                               | number |
+| pnl_usd                  | (Only applicable on trade close) The total profit and loss of the trade, in USD. | number |
```

Comment:
probably we need 2 fields (maker_pnl_usd / taker_pnl_usd)

eg : 
bluefin , liquidators (taker) will liquidate users (maker) position. 
gmx , basicially users (taker) are trading against the gm/glp pools (maker)

Reply:
yeah, agreed